libs/api/domains/driving-license/src/lib/graphql/main.resolver.ts

```diff
@@ -43,4 +44,9 @@ export class MainResolver {
   drivingLicenseRemarkTypes() {
     return this.drivingLicenseService.getRemarkTypes()
   }
+
+  @Query(() => PenaltyPointStatus)
```

Comment:
Would like to see this as a field on driving license. Nice to make a single query:
```graphql
query {
  drivingLicense {
    nationalId
    isPenaltyPointsOk
  }
}
```

Reply:
On further notice, you don't have to have a driving license to have some points in your name.